agent/src/http_server/routes/get_heaviest_files.rs

```diff
@@ -0,0 +1,11 @@
+use serde::Serialize;
+
+#[derive(Serialize)]
+struct FileInfo {
+    path: String,
+    size: u64,
+}
+
+pub async fn get_heaviest_files() -> &'static str {
+    "Get Heaviest Files"
+}
```

Comment:
As suggested by @majent4 
```suggestion
async fn heaviest_files() -> Json<Vec<FileInfo>> {
    let files: Vec<FileInfo> = vec![
        FileInfo {
            path: "/tmp/file1".to_string(),
            size: 19990,
        },
        FileInfo {
            path: "/tmp/file2".to_string(),
            size: 19000,
        },
    ];
    Json(files)
}
```